apps/website/screens/migration/Components16MigrationPage.tsx

```diff
@@ -0,0 +1,320 @@
+                      uncontrolled behavior or <Code>expanded</Code> and <Code>onExpandedChange</Code> for controlled
+                      behavior
+                    </DxcBulletedList.Item>
+                  </DxcBulletedList>
+                </DxcParagraph>
+              </>
+            ),
+          },
+          // TODO: CHECK PELAYO PR FOR THE NEW FOOTER API
+          {
+            title: "ApplicationLayout.Footer",
+            content: (
+              <DxcParagraph>
+                The <Code>Footer</Code> component no longer accepts children. It now provides two different props,{" "}
+                <Code>leftContent</Code> and <Code>rightContent</Code>, for placing custom elements in their respective
+                areas. In <Code>reduced</Code> mode, the footer will no longer display custom content on the right side,
+                instead it will display the copyright statement. A new <Code>logo</Code> prop has also been added to be
+                able to support custom branding.
+              </DxcParagraph>
+            ),
+          },
+        ],
+      },
+      {
+        title: "Badge",
+        content: (
+          <DxcParagraph>
+            Due to the recent token updates, we have aligned the <Code>color</Code> prop with a semantic color approach.
+            The available values are now <Code>'primary'</Code>, <Code>'secondary'</Code>, <Code>'tertiary'</Code>,
+            <Code>'success'</Code>, <Code>'info'</Code>, <Code>'neutral'</Code>, <Code>'warning'</Code>, and
+            <Code>'error'</Code>, replacing the previous functional options (<Code>'grey'</Code>, <Code>'blue'</Code>,{" "}
+            <Code>'green'</Code>, <Code>'orange'</Code>, <Code>'red'</Code>,<Code>'yellow'</Code>, <Code>'purple'</Code>
+            ).
+          </DxcParagraph>
+        ),
+      },
+      {
+        title: "DataGrid",
+        content: (
+          <DxcParagraph>
+            This component has exited the experimental phase and is now fully supported, featuring new functionalities
+            like on-demand loading (thanks to the <Code>childrenTrigger</Code> prop), possibility to have rows expanded
+            by default and the possibility to render any page by default. To know more about these new features, please
+            refer to the updated documentation.
+          </DxcParagraph>
+        ),
+      },
+      {
+        title: "Heading",
+        content: (
+          <DxcParagraph>
+            The weight options have been refactored, from <Code>'light' | 'normal' | 'bold'</Code> to{" "}
+            <Code>'default' | 'regular' | 'light'</Code>. Additionally, a heading level 6 is now supported.
+          </DxcParagraph>
+        ),
+      },
+      {
+        title: "Slider",
+        content: (
+          <DxcParagraph>
+            <Code>tabIndex</Code> prop has been removed. The prop was not having any effect in the component and was not
```

Comment:
I would instead change the text to:
The prop was not working as we intended, so in the meantime we decided to remove it.
I believe that at some point the component will need to set the tabIndex.